src/main/java/com/hm/achievement/listener/statistics/KillsListener.java

```diff
@@ -67,19 +69,19 @@ public void onEntityDeath(EntityDeathEvent event) {
 
 		MultipleAchievements category = MultipleAchievements.KILLS;
 
-		if (mainConfig.isConfigurationSection(category + "." + mobName)
-				&& player.hasPermission(category.toPermName() + '.' + mobName)) {
-			updateStatisticAndAwardAchievementsIfAvailable(player, category, mobName, 1);
+		Set<String> foundAchievements = Sets.newHashSet();
```

Comment:
I don't think there's any value in adding a dependency on Guava in this class. `new HashSet<>()` is actually shorter.